host/activity_test.go

```diff
@@ -220,9 +220,9 @@ func (s *integrationSuite) TestActivityHeartbeatDetailsDuringRetry() {
 						ActivityType:           &commonpb.ActivityType{Name: activityName},
 						TaskQueue:              &taskqueuepb.TaskQueue{Name: tl},
 						Input:                  nil,
-						ScheduleToCloseTimeout: timestamp.DurationPtr(4 * time.Second),
-						ScheduleToStartTimeout: timestamp.DurationPtr(4 * time.Second),
-						StartToCloseTimeout:    timestamp.DurationPtr(4 * time.Second),
+						ScheduleToCloseTimeout: timestamp.DurationPtr(8 * time.Second),
+						ScheduleToStartTimeout: timestamp.DurationPtr(8 * time.Second),
+						StartToCloseTimeout:    timestamp.DurationPtr(8 * time.Second),
```

Comment:
why?

Reply:
The test had a 4 second timeout for activity in which it needs to be retried 2 times.  First time activity sleeps for 2 seconds which results in second activity to be picked up atleast after 2/3 seconds.  Which results in second attempt to happen within a second.  
I think the reason it was passing consistently before due to the timer firing early issue because we only used the second resolution.